src/app/about/about.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, OnInit} from '@angular/core';
+import {ContactService} from '../services/contact.service';
+import {SmsinfoService} from '../services/smsinfo.service';
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styleUrls: ['./about.component.scss']
+})
+export class AboutComponent implements OnInit {
+  phoneNumber = '';
+
+  constructor(private contactService: ContactService,
+              private smsInfoService: SmsinfoService) {}
+
+  getContactEmail() {
```

Comment:
this can just be a getter `get contactEmail() { ... "` and then in template {{contactEmail}}